CORRELATION_CONTEXT_HTTP_HEADER_FORMAT.md

```diff
@@ -0,0 +1,64 @@
+# Correlation Context HTTP Header Format
+
+A correlation context header is used to pass the name-value context properties (also known as tags) for the trace. This is a companion header for the `Trace-Context`. The values are intended for control of logging systems and should be passed along to any child requests. 
```

Comment:
Rm "also known as tags"

Reply:
Rm reference to logging systems, it's unnecessary restriction

Reply:
Should it say something about whether nodes are allowed to drop these headers? If they are, should there be some recommendation which ones to drop, e.g. with longest value first?

Reply:
fixed